ui/options_metadata.ui

```diff
@@ -77,6 +77,27 @@
         </property>
        </widget>
       </item>
+      <item>
+       <widget class="QLabel" name="originaldate_label">
+        <property name="text">
+         <string>Original release date:</string>
+        </property>
+       </widget>
+      </item>
+      <item>
+       <widget class="QRadioButton" name="originaldate_use_recording">
+        <property name="text">
+         <string>Use the first release date of the recording</string>
+        </property>
+       </widget>
+      </item>
+      <item>
+       <widget class="QRadioButton" name="originaldate_use_releasegroup">
+        <property name="text">
+         <string>Use the first release date of the release group</string>
+        </property>
+       </widget>
+      </item>
      </layout>
```

Comment:
I believe that this has so much opportunity to confuse people when file naming scripts go wrong that the UI needs something more to warn / help people.